hw/dv/doc/dv_plan_template.md

```diff
@@ -19,9 +19,9 @@ applicable. Once done, remove this comment before making a PR. -->
   * Verify TileLink device protocol compliance with an SVA based testbench
 
 ## Current status
-* [Design & verification stage]({{< relref "doc/project/hw_dashboard" >}})
+* [Design & verification stage]({{< relref "hw" >}})
   * [HW development stages]({{< relref "doc/project/hw_stages.md" >}})
-* DV regression results dashboard (link TBD)
+* [Simulation results](https://reports.opentitan.org/hw/ip/foo/dv/latest/results.html)
```

Comment:
Should you make the `foo` more pronounced?  Just so people don't skip over it and use it as is?

Reply:
This file is autogenerated now with `uvmdvgen` (`hw/util/uvmdvgen/dv_plan.md.tpl`). Plus, the comment at the top of the page suggests globally searching and replacing 'foo' with the actual IP name.